theano/gpuarray/dnn.py

```diff
@@ -1282,6 +1282,11 @@ def dnn_pool(img, ws, stride=(1, 1), mode='max', pad=(0, 0)):
 
     """
     img = gpu_contiguous(img)
+    if mode == "sum":
```

Comment:
done

On Thu, Sep 8, 2016 at 12:11 PM, abergeron <notifications@github.com> wrote:

> In theano/gpuarray/dnn.py
> <https://github.com/Theano/Theano/pull/4932#discussion_r78037755>:
>
> > @@ -1282,6 +1282,11 @@ def dnn_pool(img, ws, stride=(1, 1), mode='max', pad=(0, 0)):
> >
> >      """
> >      img = gpu_contiguous(img)
> > +    if mode == "sum":
>
> Update the doc to mention sum.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Theano/Theano/pull/4932/files/4aab58b485d96dfee74701110abf6d971b9dcdaf#r78037755>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AALC-2VFcwwQ24dhHffGI2gEWKUdZIu0ks5qoDOjgaJpZM4Jz8eJ>
> .
>